### PR TITLE
Add support for moving messages to another region

### PR DESCRIPTION
**Pull request by panga**
@scottjbarr

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -13,30 +13,20 @@ import (
...
 		flag.Usage()
 		os.Exit(1)
 	}
 
-	if os.Getenv("AWS_REGION") == "" {
-		fmt.Printf("AWS_REGION not set")
-		os.Exit(1)
-	}
+	log.Printf("source queue url: %v", *src)
+	log.Printf("destination queue url: %v", *dest)
```
Comment (scottjbarr) [inline main.go]: Please remove the output lines.

File: main.go [hunk 2]
```diff
@@ -103,3 +93,7 @@ func main() {
 		wg.Wait()
 	}
 }
+
+func getRegionFromQueueURL(url string) string {
+    return strings.Split(url, ".")[1]
+}
```
Comment (scottjbarr) [inline main.go]: New line at end of file please :)

File: README.md [hunk 3]
```diff
@@ -18,9 +18,7 @@ Download the appropriate binary from the
 
 ## Configuration
 
-The `AWS_SECRET_ACCESS_KEY`, `AWS_ACCESS_KEY_ID`, and ,`AWS_REGION`
-environment variables must be set.
-
+The `AWS_SECRET_ACCESS_KEY` and `AWS_ACCESS_KEY_ID` environment variables must be set, except for EC2 instances with IAM roles.
```
Comment (scottjbarr) [inline README.md]: Please remove the reference to EC2, and keep line length < 80 chars.

File: main.go (hunk 2 above)
Comment (panga) [reply main.go]: done

**Comment by scottjbarr** [comment]
Thanks @panga !

**Comment by scottjbarr** [comment]
This has been reverted due to #9

**Pull request merged**